examples/multi_python_versions/MODULE.bazel

```diff
@@ -17,10 +17,8 @@ python.defaults(
 )
 python.toolchain(
     configure_coverage_tool = True,
-    python_version = "3.8",
-)
-python.toolchain(
-    configure_coverage_tool = True,
+    # Only set when you have mulitple toolchain versions.
+    is_default = True,
```

Comment:
Adding back `is_default = True,` in the merge commit wasn't the intention, was it? When we're using `python.defaults`, we shouldn't also use `is_default = True`.

Reply:
I probably messed up the merge conflict handling.

Reply:
Fix in #2924, which also moves other examples and docs over to `python.defaults`.